service/gitea/gitea.go

```diff
@@ -88,6 +74,8 @@ func (g *PullRequest) Post(_ context.Context, c *reviewdog.Comment) error {
 	return nil
 }
 
+func (*PullRequest) ShouldPrependGitRelDir() bool { return true }
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method PullRequest.ShouldPrependGitRelDir should have comment or be unexported
<!-- __reviewdog__:ChAzNTllZjM4YTI5ZjA0OWMxEg9nb2xpbnQtY2lyY2xlY2k= -->